### PR TITLE
Load svg files with svg-loader

### PR DESCRIPTION
**Pull request by teosz**

### CODE REVIEW

**Comment by teosz** [comment]
cc @eliperelman

File: package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
     "exports-loader": "0.6.3",
     "expose-loader": "0.7.1",
     "file-loader": "0.9.0",
+    "svg-url-loader": "^1.1.0",
```
Comment (eliperelman) [inline package.json]: Please switch this to an exact version by removing the caret.

**Comment by teosz** [comment]
You can look for benefits in the loader README.md: 
https://github.com/bhovhannes/svg-url-loader/blob/master/README.md

**Comment by eliperelman** [comment]
Thank you!

**Comment by eliperelman** [comment]
Published to npm @ 2.6.0.

**Pull request merged**